doc/subsystems/shell/shell.rst

```diff
@@ -235,13 +235,39 @@ Simple command handler implementation:
 		return 0;
 	}
 
-.. warning::
-	Do not use function :cpp:func:`shell_fprintf` or shell print macros:
-	:c:macro:`shell_print`, :c:macro:`shell_info`, :c:macro:`shell_warn`,
-	:c:macro:`shell_error` outside of the command handler because this
-	might lead to incorrect text display on the screen. If any text should
-	be displayed outside of the command context, then use the
-	:ref:`logger` or `printk` function.
+:cpp:func:`shell_fprintf` or shell print macros: :c:macro:`shell_print`,
```

Comment:
PR #11604 is making the same changes to this shell.rst file?  (I've commented on that PR).

Reply:
applied you suggestions from #11604.